pom.xml

```diff
@@ -3,9 +3,9 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>io.dropwizard.revolver</groupId>
+    <groupId>io.revolver</groupId>
     <artifactId>dropwizard-revolver</artifactId>
```

Comment:
change artifact id to dropwizard-revolver-resilience4j (instead of version)

Reply:
Changed